gnomad_qc/v4/subset.py

```diff
@@ -244,22 +244,24 @@ def main(args):
             subset_ht.count(),
         )
 
-    vds = hl.vds.filter_samples(vds, subset_ht, remove_dead_alleles=True)
+    if args.include_gru_controls:
+        gru_subset_ht = meta_ht.filter(
+            (meta_ht.project_meta.consent == "GRU") & meta_ht.project_meta.controls
```

Comment:
I'm not sure, we can ask the PMs, but given that most are captured by GRU, I think this is fine for now until we can get an answer back from them

Reply:
I just asked the PMs so hopefully we hear soon

Reply:
Christine said "depends of if the person asking for the subset has approved access for the collections - they would have to apply via dbgap to gain access and show that approval to you before you can include collections in a subset for them.  Happy to help review the request if helpful." So I think this may become a bit more complicated that we'd like and we can't implement it this way. I think we need to kick the work back up to the PMs/requester and have them include the workspace/IDs for whatever GRU controls are being asked for in the original request and they should be handling confirmation of GRU access.